routes/truckRoutes.js

```diff
@@ -1 +1,41 @@
-// put your Express router code in here
\ No newline at end of file
+var express = require('express');
+//var trucks = require('../foodTrucks');
+var router = express.Router();
+
+
+
+router.route('./trucks')
```

Comment:
Since  you've mounted this router on the `/trucks` path in your index.js file, the route you use in your router needs to be relative to that.